### DISCUSSION TITLE
Can I use mimir on Kubernetes using filesystem local storage

### DISCUSSION THREAD

**Discussion by mohamedosama58** [grafana/mimir#9064] [Help and support]
Can I use mimir on Kubernetes using filesystem local storage ? I have Kubernetes cluater with +100 microservices and I can't use any cloud provider storage. And what should i edit in values file

**Comment by pstibrany** [comment]
Hello, Mimir is designed to work with a "cloud storage", even if it's only Minio running in your local cluster. The only exception is when running Mimir in a one instance single-binary mode, but then Mimir is just more complex version of Prometheus.

If you are using provided Helm chart, you can enable use of Minio by setting `minio.enabled` field in your values.yaml.
